### PR TITLE
feat: added payment integration , order summary and order history

### PR DESCRIPTION
**Pull request by poornima-alugubelly**
User can now place an order 
User can check profile--> orders for order history

### CODE REVIEW

File: src/components/Footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+export const Footer = () => {
+	return (
+		<footer class="footer flex-column">
+			<p>Made by poornima</p>
+			<ul class=" flex-center">
+				<li>
+					<a href="https://github.com/poornima-alugubelly" target="_blank">
```
Comment (Rishi05051997) [inline src/components/Footer/Footer.jsx:7]: Instead of a tag you can Link

File: src/components/NavBar/NavBar.jsx [hunk 2]
```diff
@@ -114,28 +114,41 @@ export const NavBar = () => {
...
 			>
+				<li className="padding-s">
+					<img
+						src={IMAGES.dismissBlue}
+						className="btn-dismiss item-top-left"
+						onClick={() => setNavIsOpen(false)}
+					/>
+				</li>
+
+				{/* <i
```
Comment (Rishi05051997) [inline src/components/NavBar/NavBar.jsx:138]: remove commented code please

File: src/context/UserDataContext.js [hunk 3]
```diff
@@ -85,6 +88,24 @@ const UserDataProvider = ({ children }) => {
 					console.log("error", err);
 				}
 			})();
+		auth.isAuth &&
+			(async () => {
+				setLoading(true);
+				try {
+					const res = await getOrdersService(auth.token);
+					console.log("orders", res);
```
Comment (Rishi05051997) [inline src/context/UserDataContext.js:96]: remove console

File: src/context/UserDataContext.js [hunk 4]
```diff
@@ -85,6 +88,24 @@ const UserDataProvider = ({ children }) => {
...
+					if (res.status === 200) {
+						userDataDispatch({
+							type: SET_ORDERS,
+							payload: { orders: res.data.orders },
+						});
+
+						setLoading(false);
+					}
+				} catch (err) {
+					console.log("error", err);
```
Comment (Rishi05051997) [inline src/context/UserDataContext.js:106]: remove console

**Review by Rishi05051997 — APPROVED** [review]
Good to go, destructuring also also awesome , I left some comments please complete that

**Review by preethasri — APPROVED** [review]
good to merge👍

**Pull request merged**